pkg/networkservice/core/chainbreak/server.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package chainbreak provides wrapper chains elements to execute Request, Close on element out of the chain
+package chainbreak
```

Comment:
```suggestion
package break
```
```suggestion
package tail
```

Reply:
It is not a tail chain element, it is a chain breaker wrapper.

Reply:
Let's consider this discussion here https://github.com/networkservicemesh/sdk/pull/520#discussion_r506042985